.circleci/config.yml

```diff
@@ -381,22 +381,12 @@ jobs:
       - store_artifacts:
           path: /tmp
 
-  # Run nightly, to verify 'dep update' works
-  depupdate:
+  # Run nightly, to verify 'dep ensure' works
+  depdiff:
```

Comment:
I don't get it - why do you remove the existing job ?

Reply:
In general - removing (or renaming) existing functionality should be preceded by a discussion and done in separate PRs.

Reply:
The reason I renamed the target is to more accurately reflect what it's doing.  It's not just running `dep ensure`, it's also diffing against HEAD to verify that the dependencies are the correct ones.

My understanding that this is just a name referenced from the jobs below (i.e. the name is not significant, so long as it's used consistently throughout this file).  If that's not the case I have no issue with reverting.

Reply:
Confused: the depupdate target is supposed to do dep update (ensure --update), pulling any new dependencies. So will have diffs. 

The job is intended to verify that the toml file is valid and the most recent version of packages are valid.

You seem to remove this test - and replace it with one that only checks that the lock file matches the vendor.
Happy to see the new test. 

( unless depend.update was changed to not do an update but only ensure - in which case someone else broke it  )